status-server/src/public/js/client.js

```diff
@@ -191,6 +191,12 @@ function renderStatus(urlKey) {
         break;
     }
     document.getElementById("led"+idx).style.backgroundColor = color;
-    //led.fillRect(0,0,40,40);
+    var audio = document.getElementById('buzz');
```

Comment:
We should be able to pause/stop from a setTimeout to just play the right number of milliseconds of the clip